app/assets/stylesheets/modules/posts.scss

```diff
@@ -0,0 +1,354 @@
+.posts {
+  margin: 0 auto;
+  max-width: 960px;
+  text-align: left;
+  width: 70%;
+
+  &.for-archive {
+    box-sizing: border-box;
+    padding: 0 50px;
+
+    @include mq-mobile(){ padding: 0 20px; }
+  }
+
+  .pagination {
+    color: $blog-color;
+    margin: -2em 0 4em;
+    overflow: auto;
+    text-align: center;
+
+    .previous_page {
```

Comment:
yes, I got your point, but these class are returned by will_paginate gem, and its very hard to customize. so for this reason I use these classes